### DISCUSSION TITLE
Check if matrix has a value

### DISCUSSION THREAD

**Discussion by ghost** [josdejong/mathjs#2317] [Ideas]
Good day.
Do we have a function for simple check if a matrix has some value?
M.has(1), M.includes(1)...

**Comment by josdejong** [comment]
There is no such method. Maybe intereresting to implement. 

You can iterate over a matrix using `matrix.forEach`, and then do your test to see if a value is matching your condition. Not very efficient but here an example:

```js
function includes(matrix, value) {
  let found = false

  matrix.forEach(item => {
    if (!found && math.equal(item, value)) {
      found = true
    }
  })

  return found
}

// usage:
const m = math.matrix([[1, 2], [3, 4]])
includes(m, 3) // true
includes(m, 5) // false
includes(m, math.bignumber(3)) // true
```

**Comment by ghost** [comment]
Thank you. Good advice.
